scripting/get5.sp

```diff
@@ -846,8 +846,7 @@ public Action Command_Stop(int client, int args) {
         return Plugin_Handled;
     }
 
-    int roundsPlayed = GameRules_GetProp("m_totalRoundsPlayed");
-    if (g_GameState != GameState_Live || roundsPlayed == 0) {
+    if (g_GameState != GameState_Live || g_PendingSideSwap == true ) {
```

Comment:
Have you checked to see what happens if someone tries to use !stop the first round of a game? The plugin should be able to work (the game does generate a backup file for the start of round 0, as far as I know), but I'm not sure that it is supported correctly yet.